src/server/services/salesforce/contactmomenten.ts

```diff
@@ -52,12 +41,24 @@ export async function fetchContactmomenten(
   requestID: RequestID,
   authProfileAndToken: AuthProfileAndToken
 ) {
+  const base64encodedPK = getFromEnv('BFF_SALESFORCE_API_KEY');
```

Comment:
Het is geen API KEY maar een private key toch? Maakt het ook meteen extra duidelijk waar PK voor staat in het variabel.
```suggestion
const base64encodedPK = getFromEnv('BFF_SALESFORCE_PRIVATE_ENCRYPTION_KEY');
```